### PR TITLE
add role on toggelMore action element

### LINKED ISSUE #11195: A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_AI4D: Control type set wrong for add cell and toggle option
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#WCAG4.1.2;#WCAG1.3.1;#SQL Azure Data Studio;#FTP;#AI4D;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:WCAG4.1.2, WCAG1.3.1
Tool: Accessibility insights for desktop

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Click on cell and select code cell.
5.  Launch accessibility insights for desktop and hover over the tool bar .

**Actual:**
On hovering over the controls in the toolbar "add cell" and "toggle option" has been set as "menu item" and "list item" respectively.

**Expected:**
The control should be set as button and announced as button itself.

**User Impact:**
Screen reader users will get wrong control information.

**Attachment link for Reference:**
11195A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_AI4D

### PR DESCRIPTION
**Pull request by MaddyDev**
This PR fixes #11195 

image

### CODE REVIEW

File: src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts [hunk 1]
```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
 
 	public onInit(elementRef: HTMLElement, context: CellContext) {
 		this._moreActionsElement = elementRef;
+		this._moreActionsElement.setAttribute('aria-haspopup', 'menu');
 		if (this._moreActionsElement.childNodes.length > 0) {
 			this._moreActionsElement.removeChild(this._moreActionsElement.childNodes[0]);
 		}
-		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL });
+		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL, ariaLabel: 'More' });
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:158]: The aria label should be localized

File: src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts [hunk 2]
```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
 
 	public onInit(elementRef: HTMLElement, context: CellContext) {
 		this._moreActionsElement = elementRef;
+		this._moreActionsElement.setAttribute('aria-haspopup', 'menu');
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:154]: Where are you getting 'menu' from for the value? All the examples I see use true/false

Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:154]: Shouldn't this still have the role:button attribute?

Comment (MaddyDev) [reply src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:154]: I got this value from accessibility folks in the office hours meeting that I attended last week. 
[8/27 2:11 PM] Sarah Higley:     button aria-haspopup="menu"

Adding that is enough, we don't need to add role:button
image

**Review by alanrenmsft — APPROVED** [review]
looks good once you address the localization comment

File: src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts (hunk 1 above)
Comment (VasuBhog) [reply src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts]: Done.

File: src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts (hunk 2 above)
Comment (VasuBhog) [reply src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:154]: As per Maddy, we do not need to change this.

File: src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts [hunk 3]
```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
...
 		if (this._moreActionsElement.childNodes.length > 0) {
 			this._moreActionsElement.removeChild(this._moreActionsElement.childNodes[0]);
 		}
-		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL });
+		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL, ariaLabel: localize('moreActionsLabel', "More") });
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts:158]: Just use a single localized constant for this and the one on line 347 - no reason to duplicate strings

**Review by Charles-Gagnon — APPROVED** [review]
I still feel like this could be done better with the whole two separate dropdown implementations...but as far as fixing the issue goes it's fine (just see comment about combining the strings)

**Pull request merged**